virtual_machines.go

```diff
@@ -204,13 +207,18 @@ func (v *VirtualMachine) UnlinkDisk(diskID string, force bool) (task *Task, err
 	return NewTask(upid, v.client), nil
 }
 
-func (v *VirtualMachine) MoveDisk(disk, storage string) (task *Task, err error) {
+func (v *VirtualMachine) MoveDisk(disk string, params *VirtualMachineMoveDiskOptions) (task *Task, err error) {
 	var upid UPID
 
-	err = v.client.Post(fmt.Sprintf("/nodes/%s/qemu/%d/move_disk", v.Node, v.VMID), map[string]string{
-		"disk":    disk,
-		"storage": storage,
```

Comment:
if storage is empty does it just pick some default storage for the node?

Reply:
It's not mentioned in the doc, but if I remember well, PVE uses the same storage backend as the source volume if not specified.

Reply:
that's what i thought too